extension/src/help/openWebHelp.ts

```diff
@@ -26,11 +27,28 @@ export class WebHelpLibrary implements IJsonLoadable {
 	functions: Dictionary<WebHelpFunction> = {};
 	ambiguousFunctions: Dictionary<WebHelpFunction[]> = {};
 	enumerators: Dictionary<string> = {};	
-	year: string;
-	
+	_fallbackYear: string;
+	get year(): string {
+		// This was converted to a getter because WebHelpLibrary only loads once on start
+		// It would of required restart after changing the launch configuration to update the year
+		return this.getYearFromSettings() ?? this._fallbackYear;
+	}
+
+	// Issue #70 requested user configuration of the help year
+	//           this implements the proposed solution of extracting it from the launch configuration
+	private getYearFromSettings(): string|null {
+		let acad = getExtensionSettingString('debug.LaunchProgram');
```

Comment:
Maybe we need to add an independent setting for help? it looks like this break the [SRP ](https://en.wikipedia.org/wiki/Single-responsibility_principle)rule.

Reply:
Hi Hualin,
I agree this smells a bit, but wouldn't having a dedicated setting vs extracting it from an existing setting be pretty much the same operation? I guess I don't know how to fix the SRP problem since dealing with any setting is going to inject a dependency.

Reply:
If there is a third time we want to re-use this setting, then that is the time to split this setting to each individual settings.